<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopPage</class>
 <widget class="QWidget" name="ShopPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="BuffBtn" name="hpbtn" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="BuffBtn" name="atkbtn" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="BuffBtn" name="atpbtn" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="BuffBtn" name="spdbtn" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="BuffBtn" name="coinbtn" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="BuffBtn" name="revivebtn" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="curcoinsedit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>321</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>售价：50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuffBtn</class>
   <extends>QWidget</extends>
   <header>mybutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
